src/Form.js

```diff
@@ -1,29 +1,26 @@
-import { useState } from 'react';
-
+// form for user to select how many lines they want their poem to be
 const Form = (props) => {
 
-    const [userChoice, setUserChoice] = useState("");
-
-    const submitHandler = (e) => {
-        console.log("you submitted")
-    }
-
     return (
-        <form onSubmit={submitHandler}>
-            <label htmlFor="lineNumber">How many lines long do you want your poem to be?:</label>
+        <form onSubmit={props.submitHandler} className="wrapper">
+            <label htmlFor="lineNumber">How many lines do you want your poem to be?</label>
             <select
                 id="lineNumber"
                 name="lineNumber"
-                onChange={handleChange}
-                value={userChoice}
+                onChange={props.handleChange}
+                value={props.userChoice}
+                defaultValue={""}
+                required={true}
             >
                 <option value="" disabled>Pick one:</option>
-                <option value="two">2</option>
-                <option value="three">3</option>
-                <option value="four">4</option>
-                <option value="five">5</option>
+                <option value="2" required>2</option>
```

Comment:
Maybe it's just enough to add the required attribute to the select (the parent container of options), since it seems like it repeats itself on option with the value of 2. In case that causes problems, of course just stick with what you have since it seems to work